### ISSUE TITLE
Change "reenable code block" to "enable code block"

### ISSUE THREAD

**Issue by Polyducks** [chrismaltby/gb-studio#808]
**Is your feature request related to a problem? Please describe.**
"Reenable" is really difficult to scan for in the list of actions, and "disable/enable" are the expected toggle opposites. It also implies that once a thing has been reenabled that the disabled button should then read redisable, then rereenable etc etc

**Describe the solution you'd like**
Change "reenable" to "enable".

**Describe alternatives you've considered**
"Toggle code block" is another option which means the command is always called the same thing. However, enable/disable is more explicit.

**Comment by maxoakland** [comment]
I agree. Also for "Reenable" used in other areas like "Reenable Else"
